src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -16,29 +16,50 @@
+        <PropertyGroup>
+            <ExtensionsVersion>6.0.0</ExtensionsVersion>
+            <SystemVersion>6.0.0</SystemVersion>
+        </PropertyGroup>
+    </Otherwise>
+  </Choose>
   <ItemGroup>
     <PackageReference Include="Jil" Version="3.0.0-alpha2" />
     <PackageReference Include="MessagePack" Version="1.9.11" />
     <PackageReference Include="MessagePackAnalyzer" Version="1.7.3.7" />
-    <PackageReference Include="Microsoft.Extensions.Configuration" Version="5.0.0-rc.1.20417.14" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Xml" Version="5.0.0-rc.1.20417.14" />
-    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="5.0.0-rc.1.20417.14" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="5.0.0-rc.1.20417.14" />
-    <PackageReference Include="Microsoft.Extensions.Http" Version="5.0.0-rc.1.20417.14" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0-rc.1.20417.14" />
-    <PackageReference Include="Microsoft.Extensions.Logging.EventSource" Version="5.0.0-rc.1.20417.14" />
-    <PackageReference Include="Microsoft.Extensions.Primitives" Version="5.0.0-rc.1.20417.14" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="$(ExtensionsVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Xml" Version="$(ExtensionsVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="$(ExtensionsVersion)" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="$(ExtensionsVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Http" Version="$(ExtensionsVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="$(ExtensionsVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Logging.EventSource" Version="$(ExtensionsVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Primitives" Version="$(ExtensionsVersion)" />
     <PackageReference Include="protobuf-net" Version="3.0.73" />
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="5.0.0-rc.1.20417.14" />
-    <PackageReference Include="System.Drawing.Common" Version="5.0.0-rc.1.20417.14" />
-    <PackageReference Include="System.Formats.Cbor" Version="5.0.0-rc.1.20417.14" Condition="'$(TargetFramework)' == 'net5.0' or '$(TargetFramework)' == 'net6.0' or '$(TargetFramework)' == 'net7.0'" />
-    <PackageReference Include="System.IO.Pipelines" Version="5.0.0-rc.1.20417.14" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="$(SystemVersion)" />
+    <PackageReference Include="System.Drawing.Common" Version="$(SystemVersion)" />
+    <PackageReference Include="System.Formats.Cbor" Version="$(SystemVersion)" Condition="'$(TargetFramework)' == 'net5.0' or '$(TargetFramework)' == 'net6.0' or '$(TargetFramework)' == 'net7.0'" />
```

Comment:
These will need updates when we move to net8.0. Would it be better/possible to use this function instead: `Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net5.0'))">`

Reply:
@danmoseley Good idea, I was not aware of `IsTargetFrameworkCompatible` existence. Is it supported by older SDKs? (3.1 for example?)

Reply:
It looks like it would be 6.0 onwards , judging by the date :|
https://github.com/dotnet/msbuild/commit/51e8dd25010de68b07047ac49254e29d2af41241#